### ISSUE TITLE
cifar10 parameters setting

### ISSUE THREAD

**Issue by EvanLiu1** [www0wwwjs1/Matrix-Capsules-EM-Tensorflow#47]
@www0wwwjs1   Hi Prof. Suofei! I train CapsNet on cifar-10,  but the accuracy is 50% after 50 epochs. Could you please give me some advice about parameter setting？

Thank you very much!
